app/src/main/assets/extensions/fxr_youtube/main.js

```diff
@@ -133,7 +134,7 @@ class YoutubeExtension {
             return; // Only override projection in the Youtube watching page.
         }
         const qs = new URLSearchParams(window.location.search);
-        if (qs.get(VIDEO_PROJECTION_PARAM)) {
+        if (qs.get(VIDEO_PROJECTION_PARAM) && !qs.get(VIDEO_PROJECTION_INFERRED_PARAM)) {
```

Comment:
What about registering a navigation handler which removes the params on navigation forcing us to recompute the projection?

Reply:
Okay, let me try. Not sure if that way will 100% work